static/core/css/base.css

```
@@ -768,13 +768,65 @@ div.card-reference > div.card-body{
     min-height: 455px;
 }
 
+@media all {
+    .blue-card-margin {
+        margin: 25px 0 0 40px;
+    }
+}
```

Whats the point of the `@media` query here?
```suggestion
.blue-card-margin {
    margin: 25px 0 0 40px;
}
```